src/vm/ceeload.cpp

```diff
@@ -7722,8 +7722,8 @@ void Module::ExpandAll(DataImage *image)
     // Record references to all of the hot methods specifiled by MethodProfilingData array
     // We call MethodReferencedByCompiledCode to indicate that we plan on compiling this method
     //
-    CORBBTPROF_TOKEN_INFO * pMethodProfilingData = profileData->GetTokenFlagsData(MethodProfilingData);
-    DWORD                   cMethodProfilingData = profileData->GetTokenFlagsCount(MethodProfilingData);
+    CORBBTPROF_TOKEN_INFO * pMethodProfilingData = profileData ? profileData->GetTokenFlagsData(MethodProfilingData) : NULL;
```

Comment:
I would be better to have just one `if (profileData != NULL)` that protects the two loops below instead of checking every time `profileData` is accessed.